packages/core/src/tools/error.spec.ts

```diff
@@ -71,3 +71,25 @@ describe('formatUnknownError', () => {
     expect(formatted.message).toEqual('Uncaught {"foo":"bar"}')
   })
 })
+
+describe('createHandlingStackTrace', () => {
+  let stackTrace: StackTrace
+  function rumInstrumentationCall() {
+    stackTrace = createHandlingStackTrace()
+  }
+  function userCallTwo() {
+    rumInstrumentationCall()
+  }
+  function userCallOne() {
+    userCallTwo()
+  }
+
+  it('should get handling stack trace without instrumental function calls', () => {
+    userCallOne()
+
+    const formattedStackTrace = toStackTraceString(stackTrace)
```

Comment:
I'm not sure about it. I tried to be consistent with the way we handle the "normal" stack trace.
https://github.com/DataDog/browser-sdk/blob/1f8b7c6e258cd5cd14817c9cd55cd086f79a0af3/packages/core/src/domain/automaticErrorCollection.ts#L57-L65
Also, by looking at error.ts, it seems that the idea was to keep stack traces as  `StackTrace` type and to be able to make transformation on them.
https://github.com/DataDog/browser-sdk/blob/1f8b7c6e258cd5cd14817c9cd55cd086f79a0af3/packages/core/src/tools/error.ts#L39-L43

Maybe I should change the test to only test the stack trace as `StackTrace` type and not as string. It would also make sens because the `toStackTraceString` has already been tested.

Reply:
`StackTrace` is a tracekit type, it seems ok to me to keep it for tracekit API output but I'd be in favor of avoiding to use it internally if we can.

Reply:
Okay done